packages/styles/scss/components/tableofcontents/index.scss

```diff
@@ -3,6 +3,6 @@
 
 .#{$prefix}--tableofcontents--g100 {
   @include carbon--theme($carbon--theme--g100) {
-    @include tableofcontents;
+    @include themed-items;
   }
 }
```

Comment:
@RaphaelAmadeu this wrapper can actually go away, since we have shifted to creating the theme modifiers as a modifier class and not a parent class. 

cc: @annawen1

Reply:
Right if you take a look at https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles/scss/patterns/usecases/_usecases.scss

and https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles/scss/patterns/usecases/index.scss